src/Components/ActivationKeys/blank-state.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
in order to be consistent with our other apps, can we use `emptyState` here instead of blank state? And move the folder out to the `../Components/emptyState` location?

Or is it better to keep the empty state encapsulated in just our `ActivationKeys` section?

What I'm thinking is this:

in `src/Components/ActivationKeys/ActivationKeys.js`

```javascript
import NoActivationKeysFound from '../emptyState/NoActivationKeysFound';
...
  } else if (!isLoading && !error && !data.length) {
    pageContent = (
      <NoActivationKeysFound handleModalToggle={handleModalToggle} />
    );
```

`Components/emptyState` has

```javascript
// Components/emptyState/index.js
import NoActivationKeysFound from './NoActivationKeysFound';

export { NoActivationKeysFound };
```

```javascript
// Components/emptyState/NoActivationKeysFound.js
// everythying from this blank-state.js file but the component is just named const NoActivationKeysFound
```

This doesn't have to be done this way, what do you think?